apps/brownfield-tester/ios/BrownfieldTester/ExpoViewController.swift

```diff
@@ -0,0 +1,53 @@
+import Expo
+import React
+import ReactAppDependencyProvider
+
+class ExpoViewController: UIViewController {
```

Comment:
Would it be an idea (it might already be the plan) to move this to an Expo module?

Maybe even the ExpoViewRepresentable/ExpoView as well?